### ISSUE TITLE
onStart children elements exhausts browser

### ISSUE THREAD

**Issue by ConduciveMammal** [aholachek/animate-css-grid#25]
I'm trying to use `onStart`/`onEnd` to add/remove a class from the grid's children (About 20 in length), but even just `onStart` exhausts my browser and causes it to hang.

My JS:

```js
const { unwrapGrid, forceGridAnimation } = wrapGrid(collectionGrid, {
  duration: 250,
  stagger: 10,
  onStart: (elements) => {
    for (let i = 0; i < elements.length; i++) {
      const element = elements[i];
      element.classList.add('transitioning')
    }
  }
});
```

**Comment by aholachek** [comment]
Hi, I think what's happening is that adding a class to elements inside the grid is triggering the mutation listener to try to animate the grid, which is starting the cycle over again into an infinite loop.

This is definitely a bug, thank you for flagging it. I'll comment here when it's fixed, hopefully by this weekend.

**Comment by ConduciveMammal** [comment]
Thanks for your reply!

I found, also that doing a very simple

```js
onStart: collectionGrid.classList.add(‘transitioning’);
```

Caused the same issue to occur.

**Comment by aholachek** [comment]
Hi @ConduciveMammal, I've just released v1.4.3, please give it a try when you get a chance and let me know if it resolves the problem.
